packages/frontend/src/components/Trade/Long/index.tsx

```diff
@@ -532,9 +522,9 @@ const CloseLong: React.FC<BuyProps> = ({
       if (longSqthBal.lt(amount)) {
         closeError = 'Insufficient oSQTH balance'
       }
-      if (amount.gt(balance)) {
-        openError = 'Insufficient ETH balance'
-      }
+      // if (amount.gt(balance)) {
```

Comment:
why comment this out?

Reply:
openError is not being used in this component